internal/dao/dao_interfaces.go

```diff
@@ -73,6 +73,9 @@ type ReservationDao interface {
 	// GetById returns reservation for a particular account.
 	GetById(ctx context.Context, id int64) (*models.Reservation, error)
 
+	// GetAWSById returns reservation for a particular account.
+	GetAWSById(ctx context.Context, id int64) (*models.AWSReservation, error)
+
```

Comment:
Maybe we should also add tickets for creating those methods for GCP and Azure?

Reply:
Yeah, well, we will eventually figure this out when implementing these features :-)